### DISCUSSION TITLE
MagickSettings: why options backed by Drawing are not all handled in the same way?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1520] [General] [answered]
:wave: 

Looking at MagickSettings, I'm trying to understand why there is different way of handling the properties both backed in `Drawing` and `_options`:

As example:

- the getter for `FillColor` is backed directly from the `Drawing` property.
	https://github.com/dlemstra/Magick.NET/blob/56764717217ce29705c13168cbea815d14073d92/src/Magick.NET/Settings/MagickSettings.cs#L122-L130

- the getter for `FontFamily` is looking into options
	https://github.com/dlemstra/Magick.NET/blob/56764717217ce29705c13168cbea815d14073d92/src/Magick.NET/Settings/MagickSettings.cs#L166-L174

Is this intentional? And why?

Thanks.

Regards.

**Comment by dlemstra** [comment, marked as answer]
I have no idea why I did that but it makes more sense to return `Drawing.FontFamily`. I really appreciate all your questions about my library and the research you are doing. Looking forward to another PR from you to fix his!

**Reply by Gounlaf** [reply]
Here it is https://github.com/dlemstra/Magick.NET/pull/1521

**Reply by Gounlaf** [reply]
> I have no idea why I did that [...]

I thank it was to avoid make low level/native api calls, but... nope ; the Drawing class is created with the native api, but that's all.
